domain/schema/model/sql/0015-charm.sql

```diff
@@ -629,7 +629,5 @@ SELECT
     c.reference_name,
     c.revision,
     c.source_id,
-    c.architecture_id,
-    cm.name
-FROM charm AS c
-JOIN charm_metadata AS cm ON c.uuid = cm.charm_uuid;
+    c.architecture_id
+FROM charm AS c;
```

Comment:
This view now provides no value when using sqlair.

`SELECT &thing.* from v_charm_locator` is now equivalent to `SELECT &thing.* FROM charm`

Reply:
Follow up PR.